pwiz_tools/Skyline/Util/Adduct.cs

```diff
@@ -1363,33 +1366,43 @@ public ParsedMolecule ApplyToMolecule(ParsedMolecule molecule)
                 {
                     resultDict[element] *= MassMultiplier;
                 }
+                monoMassOffset *= MassMultiplier;

```

Comment:
This seems like it would impact molecules that are not mass-only, but have both a formula and a mass offset. Am I right about that? If so, I am not seeing a test for that case.

Reply:
Ooh, excellent point. That wasn't properly implemented, nor tested. Set now.